docs/migration-guide.md

```diff
@@ -2,11 +2,196 @@
 
 ## Contents
 
+- [Neutrino v8 to v9](#neutrino-v8-to-v9)
 - [Neutrino v7 to v8](#neutrino-v7-to-v8)
 - [Neutrino v6 to v7](#neutrino-v6-to-v7)
 - [Neutrino v5 to v6](#neutrino-v5-to-v6)
 - [Neutrino v4 to v5](#neutrino-v4-to-v5)
 
+## Neutrino v8 to v9
+
+Neutrino v9 is our largest release ever, bringing the preset and middleware ecosystem back to the
+native tools and utilities for which they were originally created. The biggest breaking change
+is the necessity to use external tools *alongside* Neutrino now, i.e. webpack, ESLint, Jest, Karma,
+and others' native CLIs will be used in tandem with Neutrino. A list of changes is detailed below for
+migrating:
+
+- **BREAKING CHANGE** The minimum supported Node.js version is v8.3. Node.js v9 is no longer supported
+[#792](https://github.com/neutrinojs/neutrino/pull/792).
+- **BREAKING CHANGE** Consumption of middleware and presets are now dependent on the external CLIs and
+configuration files for which they are intended [#852](https://github.com/neutrinojs/neutrino/pull/852).
+This does not negate the need for a `.neutrinorc.js` file
+in your projects, but rather is the configuration format for declaring all the middleware for a project. As
+an example, using the `@neutrinojs/react` preset also requires an installation of `webpack`,
+`webpack-dev-server`, and `webpack-cli`. You can then add a `webpack.config.js` file to the project which
+will tell webpack to use the Neutrino middleware to load its configuration:
+
+```js
+// webpack.config.js
+const neutrino = require('neutrino');
+
+module.exports = neutrino().webpack();
+```
+
+Or for ESLint:
```

Comment:
The `.eslintrc.js` is something that many people might have already created - they'll need to update it, given the Neutrino API changes